baseplate/__init__.py

```diff
@@ -1,13 +1,19 @@
 import os
 import sys
 
+from typing import Dict, TYPE_CHECKING
+
 from baseplate import config, metrics
 from baseplate.core import Baseplate
 from baseplate.diagnostics import tracing
 from baseplate._utils import warn_deprecated
 
 
-def metrics_client_from_config(raw_config):
+if TYPE_CHECKING:
+    import raven
+
+
+def metrics_client_from_config(raw_config: Dict[str, str]) -> metrics.Client:
```

Comment:
The funding instrument service saw this `Dict[str, str]` show up in a lot of places, so we decided to define a variable for it to make it more obvious to the reader. I'm not sure if I like this pattern or not, but I'm interested to get your thoughts. I think it's all the same to mypy.

https://github.snooguts.net/reddit/reddit-service-funding-instruments/blob/1d490bbd9c79d0329f34884dcc353ab8d413a805/funding_instruments/common_types.py#L4
https://github.snooguts.net/reddit/reddit-service-funding-instruments/blob/1d490bbd9c79d0329f34884dcc353ab8d413a805/funding_instruments/models/sql.py#L7

Reply:
Yeah! Then you can just import those type defs too. That should be really nice. I'll get some aliases going.

Reply:
+1